### PR TITLE
Allow 'bind' in config to specify bind address

### PR DESCRIPTION
**Pull request by mrvdb**
Minimal changes, definitely WIP

To resolve:
- how to support dualstack when not using localhost?
- net/http package uses string, mentions IP address instead of bind,
  need info.

For issue #5

### CODE REVIEW

File: app.go [hunk 1]
```diff
@@ -404,6 +404,10 @@ func Serve() {
 	http.Handle("/", r)
 
 	// Start web application server
+	var bindAddress = app.cfg.Server.Bind
+	if bindAddress == "" {
+		bindAddress = "localhost"
+	}
 	if app.cfg.IsSecureStandalone() {
 		log.Info("Serving redirects on http://localhost:80")
```
Comment (thebaer) [inline app.go]: We'll probably want to include the `bindAddress` here too.

**Comment by thebaer** [comment]
Besides the one mentioned change, this looks good. I'm not sure about the net/http stuff off the top of my head, and otherwise for testing I'd like to defer to you or anyone else with a dualstack config. I just don't have the setup to experiment and figure out the best way forward there.

File: app.go (hunk 1 above)
Comment (mrvdb) [reply app.go]: Yeah, was in doubt about that one. pushed

**Comment by mrvdb** [comment]
I've read up a bit on the methods. The `net/http/Listen` method is basically the same as `net/Listen` with the network set to `tcp`, meaning dualstack tcp.
In practice this means if you use `localhost` the bind will probably go to ipv6 instead of ipv4 (which is correct)
So, we're good to go I think. The only thing to be aware of is the single binding limitation. For example there is no way to bind it to localhost on ipv4 and ipv6 simultaneously. The library is a bit higher level than I thought.

**Comment by BenOvermyer** [comment]
This looks fine to me.

We should really get some integration tests (e.g., InSpec) set up to automatically test variations of this config, but that is probably outside the scope of this PR.

**Comment by thebaer** [comment]
Okay great, I'll merge this now.

**Pull request merged**